src/site/content/en/lighthouse-performance/third-party-summary/index.md

```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: Reduce the impact of third-party code
+description: |
+  How third-party code like advertising networks and analytics services affect page load performance,
+  and how to optimize third-party code.
+web_lighthouse:
+  - third-party-summary
+---
+
+To add an advertising network, social media button, A/B test, or analytics service to your page,
+you usually need to add a third-party script to your HTML. These [third-party scripts can
+significantly affect your page load performance](/third-party-javascript/). Lighthouse flags pages
+that have too much third-party code:
+
+<figure class="w-figure">
+  <img class="w-screenshot w-screenshot--filled" src="third-party-summary.png" 
+       alt="A screenshot of the 'Reduce the impact of third-party code' audit.">
+  <figcaption class="w-figcaption">
+    <b>Reduce the impact of third-party code</b>
+  </figcaption>
+</figure>
+
+## How the Lighthouse audit for third-party code fails
+
+A third-party script is any script hosted on a domain that's different than the domain of the URL
+that you audited with Lighthouse. As the page loads, Lighthouse calculates how long each of the
+third-party scripts blocks the main thread. This audit fails when the total blocking time is
```

Comment:
Maybe link--or define, pending the definition popup widget--"main thread"? (I'm working under the assumption that LH users may not have as robust an understanding of web dev concepts as users who access web.dev through other entry points. But that may not be a good assumption...)

Reply:
Done